src/main/java/mdlaf/components/titlepane/MaterialTitlePaneUI.java

```diff
@@ -515,13 +517,14 @@ protected void paintComponent(Graphics g) {
                 }
                 titleW = rect.x - xOffset - 4;
                 theTitle = SwingUtilities2.clipStringIfNecessary(rootPane, fm, theTitle, titleW);
+                xOffset -= SwingUtilities2.stringWidth(rootPane, fm, theTitle);
```

Comment:
@mikera This line inside your code create this bug https://github.com/vincenzopalazzo/material-ui-swing/issues/124

Please can you remove it?

Reply:
Done